kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/food2door/InformationService.java

```diff
@@ -0,0 +1,7 @@
+package com.kodilla.good.patterns.food2door;
+
+public interface InformationService {
+
+    void sendOrderInformation(Order order);
+    void failedOrderInformation(Order order, ProductDatabase productDatabase);
```

Comment:
na podstawie `Order` & `ProductDatabase` wysyła informację o powodzie tej fatalnej porażki.

Reply:
`ProductDatabase` jest serwisem, którego raczej się nie powinno przekazywać przez metody. Powinien być on wstrzyknięty podczas tworzenia klasy InformationService. Sam `Order` wystarczy

Reply:
Zaakceptowałem, ale możesz to poprawić w wolnej chwili, bo to mała rzecz